frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,84 @@
+import { click } from '@console/shared/src/test-utils/utils';
+import { waitFor } from '../../utils/helpers';
+
+const btnCreate = $('.pf-m-primary');
+const namespaceLink = $('#openshift-storage-link');
+const filterItem = $('input[data-test-id="item-filter"]');
+const OCS_OPERATOR = $('a[data-test-operator-row="OpenShift Container Storage"]');
+
+const LABEL_KEY = 'cluster.ocs.openshift.io/openshift-storage';
+const createLink = $('.pf-c-card__footer a');
+const sizeDropdown = $('button[id="ocs-service-capacity-dropdown"]');
+const optionSmallSize = $('button[id="512Gi-link"]');
+
+describe('Testing OCS Subscription', () => {
+  beforeAll(() => {
+    execSync(
+      'kubectl apply -f https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/deploy-with-olm.yaml',
+    );
+  }, 2 * MINUTE);
+
+  // Once other tests are pushed remove this
+  afterAll(() => {
+    execSync(`kubectl delete storageclusters ocs-storagecluster -n openshift-storage`);
+    execSync(
+      'kubectl delete -f https://raw.githubusercontent.com/openshift/ocs-operator/master/deploy/deploy-with-olm.yaml',
+    );
+  }, 2 * MINUTE);
+
+  it(
+    'Test Installation for OCS operator suceeds or not',
+    async () => {
+      await goToInstalledOperators();
+      await click(namespaceDropdown);
+      await click(namespaceLink);
+      await filterItem.sendKeys('OCS');
+      await waitFor(ocsOperatorStatus, 'Succeeded', 5);
+      await click(OCS_OPERATOR);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      await browser.wait(until.visibilityOf(createLink), 10 * SECOND);
+      const storageClusterLink = await getStorageClusterLink();
+      await click(storageClusterLink);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      const nodes = await selectWorkerRows();
+      await click(sizeDropdown);
+      await click(optionSmallSize);
+      await browser.wait(until.elementToBeClickable(btnCreate));
+      await click(btnCreate);
+      await browser.sleep(5 * SECOND);
+
+      // defualt sc
+      const defaultSC = execSync(`oc get storageClasses | gawk '{print $1 $2}' | grep default`);
+      const storageCR = JSON.parse(
+        execSync(`kubectl get storageclusters ocs-storagecluster -n ${NS} -o json`).toString(),
+      );
+      // Tests for storage class request is correct
+      expect(
+        defaultSC.includes(
+          storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.storageClassName,
+        ),
+      ).toBe(true);
+      // Tests for storage class
+      expect(
+        storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.resources?.requests
+          ?.storage,
+      ).toEqual('512Gi');
+      nodes.forEach((node) => {
+        const nodeJSON = JSON.parse(execSync(`oc get nodes ${node}`).toString());
+        const labelKeys = Object.keys(nodeJSON.metadata.labels);
+        expect(labelKeys).toContain(LABEL_KEY);
```

Comment:
You also need to add test for checking `schedule-able` nodes(as we have discussed) as while selecting the nodes, you haven't checked for it.

Reply:
ack